sympy/physics/mechanics/body.py

```diff
@@ -8,64 +8,90 @@
 
 class Body(RigidBody, Particle):
     """
-    Body is a common representation of RigidBody or a Particle.
-
-    A Body represents either a rigid body or particle in classical mechanics.
-    Bodies have a body-fixed reference frame, a mass, a mass center and
-    possibly a body-fixed inertia.
+    Body is a common representation of either a RigidBody or a Particle SymPy
+    object depending on what is passed in during initialization. If a mass is
+    passed in and central_inertia is left as None, the a Particle object is
+    created. Otherwise a RigidBody object will be created.
+
+    The attributes that Body possesses will be the same as a Particle instance
+    or a Rigid Body instance depending on which was created. Additional
+    attributes are listed below.
+
+    Attributes
+    ==========
+
+    name : string
+        The body's name
+    masscenter : Point
+        The point which represents the center of mass of the rigid body
+    frame : ReferenceFrame
+        The reference frame which the body is fixed in
+    mass : Sympifyable
+        The body's mass
+    inertia : (Dyadic, Point)
+        The body's inertia around its center of mass. This attribute is specific
+        to the rigid body form of Body and is left undefined for the Particle
+        form
+    loads : iterable
+        This list contains information on the different loads acting on the
+        Body. Forces are listed as a (point, vector) tuple and torques are
+        listed as (reference frame, vector) tuples.
 
     Parameters
-    ----------
-    name: String
-        Defines the name of the body. It is used as the base for defining body
-        specific properties.
+    ==========
+
+    name : String
+        Defines the name of the body. It is used as the base for defining
+        body specific properties.
     masscenter : Point, optional
-        A point that represents the center of mass of the body or particle. If no
-        point is given, a point is generated.
-    frame : ReferenceFrame (optional)
-        The ReferenceFrame that represents the reference frame of the body. If
-        no frame is given, a frame is generated.
+        A point that represents the center of mass of the body or particle.
+        If no point is given, a point is generated.
     mass : Sympifyable, optional
-        A Sympifyable object which represents the mass of the body. if no mass
-        is passed, one is generated.
-    body_inertia : Dyadic
+        A Sympifyable object which represents the mass of the body. If no
+        mass is passed, one is generated.
+    frame : ReferenceFrame, optional
+        The ReferenceFrame that represents the reference frame of the body.
+        If no frame is given, a frame is generated.
+    central_inertia : Dyadic, optional
         Central inertia dyadic of the body. If none is passed while creating
         RigidBody, a default inertia is generated.
```

Comment:
This differs than the above inertia description. I think central inertia is a reasonable choice to pass in, but the above had (Dyadic, Point) which is the inertia about an arbitrary point.